packages/extension/src/localStore/cookieStore/cookieStore.ts

```diff
@@ -17,8 +17,8 @@
  * Internal dependencies.
```

Comment:
@amovar18 I think a new directory for each store is not required because currently a single file is enough for preferences